pkg/providers/snow/reconciler/reconciler.go

```diff
@@ -0,0 +1,18 @@
+package reconciler
+
+import (
+	"context"
+
+	anywherev1 "github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/controller"
+)
+
+type SnowReconciler struct{}
+
+func NewSnowReconciler() *SnowReconciler {
+	return &SnowReconciler{}
+}
+
+func (v *SnowReconciler) Reconcile(ctx context.Context, cluster *anywherev1.Cluster) (controller.Result, error) {
```

Comment:
this is already in the snow (sub) package
```suggestion
type Reconciler struct{}

func New() *Reconciler {
	return &Reconciler{}
}

func (r *Reconciler) Reconcile(ctx context.Context, cluster *anywherev1.Cluster) (controller.Result, error) {
```